xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -320,6 +321,7 @@ struct UserProfileDetailModel: ModelProtocol {
             model.initialTabIndex = initialTabIndex
             // We might be passed some basic profile data
             // we can use this in the loading state for a preview
+            model.address = address
```

Comment:
Ensures that even if we fail to load the user's profile we can still render the omnibus with the intended navigation address.